lib/Map/RegionProvider.js

```diff
@@ -116,6 +117,17 @@ var RegionProvider = function(regionType, properties) {
      * @type {Object}
      */
     this._idIndex = {};
+
+    /**
+     * The URL of a pre-generated JSON file containing just a long list of IDs for a given
+     * layer attribute, in the order of ascending feature IDs (fids). If defined, it will
+     * be used in preference to requesting those attributes from the WFS server.
+     * @type {String}
+     */
+    this.regionIdList = properties.regionIdList;
+
+
+
```

Comment:
OIC, documenting the properties as constructor parameters rather than on the individual statements that set them. I like it - but I think the style I followed is the one we mostly use. Thoughts, @kring ?